student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age:, name: 'Unknown', parent_permission: true, classroom: nil)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] To avoid any argument error, Kindly pass the parameter to the super-initialize in the same way they are defined in the Person class like:
```ruby
super(age, name, parent_permission: parent_permission)
```